sql/database.sql

```diff
@@ -10765,3 +10765,77 @@ CREATE TABLE `patient_birthday_alert` (
   `turned_off_on` date NOT NULL,
   PRIMARY KEY  (`pid`,`user_id`)
 ) ENGINE=InnoDB;
+
+-----------------------------------------------------------
+--
+-- Table structure for table `morphine_eq`
+--
+
+DROP TABLE IF EXISTS `morphine_eq`;
+CREATE TABLE IF NOT EXISTS `morphine_eq` (
+  `id` int(11) NOT NULL AUTO_INCREMENT,
+  `datetime` datetime NOT NULL,
+  `encounter` int(10) NOT NULL,
+  `last` int(6) NOT NULL,
+  `next` int(6) NOT NULL,
+  `notes` varchar(90) NOT NULL,
+  `pid` int(6) NOT NULL,
+  PRIMARY KEY (`id`)
+) ENGINE=InnoDB;
+
+-------------------------------------------------------------
+
+--
+-- Table structure for table `morphine_list`
+--
+
+DROP TABLE IF EXISTS `morphine_list`;
+CREATE TABLE IF NOT EXISTS `morphine_list` (
+  `id` int(11) NOT NULL AUTO_INCREMENT,
+  `drugname` varchar(90) NOT NULL,
+  `multiplier` decimal(5,2) NOT NULL,
+  `days` int(3) NOT NULL,
+  PRIMARY KEY (`id`)
+) ENGINE=InnoDB;
+
+--
+-- Dumping data for table `morphine_list`
+--
+
+INSERT INTO `morphine_list` (`id`, `drugname`, `multiplier`, `days`) VALUES
```

Comment:
Very nice seed data.

Reply:
I thought it would be a good start for a practice that has patients to get them up and running right away. This way it gives them ideas and has the common drugs in the system already. Less work.

Reply:
Great idea